packages/block-library/src/cover/edit.js

```diff
@@ -297,7 +299,7 @@ class CoverEdit extends Component {
 					) }
 					<div className="wp-block-cover__inner-container">
 						<InnerBlocks
-							template={ INNER_BLOCKS_TEMPLATE }
+							template={ innerBlockCount === 0 ? INNER_BLOCKS_TEMPLATE : null }
```

Comment:
Interesting, makes me wonder if this should be something to handle in `InnerBlocks` directly and not at the block level. cc @jorgefilipecosta

Reply:
Interesting the fact that this PR fixes the problem seems to indicate that we have a bug as if InnerBlocks existed the template should not have been used by InnerBlocks. I will look deeper into this problem.